log4j2.xml

```diff
@@ -6,7 +6,7 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Logger name="io.shiftleft.overflowdb" level="warn" />
+        <!-- <Logger name="flatgraph" level="warn" /> -->
```

Comment:
Wondering about this commented entry - should it be uncommented?

Reply:
no, this can stay AFAICS - the old entry was required to silence some of overflowdb's noisy log statements, and in case we want to do the same for flatgraph in future this might be useful 🤷🏻